db/test/unit/functions/receive_applications_test.sql

```diff
@@ -0,0 +1,66 @@
+begin;
+select plan(4);
+
+select has_function(
+  'ccbc_public', 'receive_applications',
+  'Function receive_applications should exist'
+);
+
+-- Test setup - user
+set jwt.claims.sub to '11111111-1111-1111-1111-111111111111';
+
+delete from ccbc_public.intake;
+delete from ccbc_public.application;
+delete from ccbc_public.application_status;
+
+insert into
+  ccbc_public.intake(id, open_timestamp, close_timestamp, ccbc_intake_number)
+overriding system value
+values
+  (1, '2022-08-19 09:00:00 America/Vancouver','2022-11-06 09:00:00 America/Vancouver', 1),
+  (2, '2021-08-19 09:00:00 America/Vancouver','2021-11-06 09:00:00 America/Vancouver', 2);
+
```

Comment:
We should add the `mocks.set_mocked_time_in_transaction` function here as the tests will start to fail after Nov 6